tests/rptest/services/redpanda.py

```diff
@@ -1819,20 +1819,29 @@ def save_executable(self):
             self._saved_executable = True
             self._context.log_collect['executable', self] = True
 
-    def search_log(self, pattern):
+    def search_log(self, pattern: str, node_matches: int = 1):
```

Comment:
you might consider changing this to be `search_log(self, pattern, nodes = self.nodes)` which gives a lot more control to the caller.

Reply:
Done.